airframe/src/main/scala/wvlet/airframe/AirframeSession.scala

```diff
@@ -353,6 +353,11 @@ private[airframe] class AirframeSession(
 
     val result =
       obj.getOrElse {
+        // strict mode
+        if (design.designOptions.disableImplicitInstanceGeneration.contains(true)) {
+          throw new MISSING_DEPENDENCY(seen, sourceCode)
```

Comment:
Does `seen` contains the current target `tpe`?

Reply:
Thanks! Fixed.